distro/run/core/pom.xml

```diff
@@ -77,6 +77,13 @@
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-webmvc</artifactId>
+      <exclusions>
+        <!-- This artifact is also excluded by the Spring Boot starter for Tomcat -->
+        <exclusion>
+          <artifactId>tomcat-annotations-api</artifactId>
+          <groupId>org.apache.tomcat</groupId>
+        </exclusion>
+      </exclusions>
```

Comment:
Did you mean to put this exclusíon at the `tomcat-embed-core` artifact?

Reply:
Oh yes, that's a copy-paste mistake.